my-app/src/app/modules/login-page/login-page.module.ts

```diff
@@ -0,0 +1,20 @@
+import {NgModule} from '@angular/core';
+import {CommonModule} from '@angular/common';
+
+import {FormsModule} from "@angular/forms";
+
+import {AuthService} from "@core/services/auth.service";
+
+import {LoginPageComponent} from "./login-page.component";
+
+@NgModule({
+  declarations: [LoginPageComponent],
+  imports: [
+    CommonModule, FormsModule
+  ],
+  exports: [LoginPageComponent],
+  providers: [AuthService]
```

Comment:
AuthService is used in many places across the app, it should belong to the Core module IMO